mappings/net/minecraft/block/StairsBlock.mapping

```diff
@@ -7,7 +9,45 @@ CLASS bru net/minecraft/block/StairsBlock
 	FIELD d WATERLOGGED Lbwb;
 	FIELD e TOP_SHAPE Lcsp;
 	FIELD f BOTTOM_SHAPE Lcsp;
+	FIELD g BOTTOM_NORTH_WEST_CORNER_SHAPE Lcsp;
+	FIELD h BOTTOM_SOUTH_WEST_CORNER_SHAPE Lcsp;
+	FIELD i TOP_NORTH_WEST_CORNER_SHAPE Lcsp;
+	FIELD j TOP_SOUTH_WEST_CORNER_SHAPE Lcsp;
+	FIELD k BOTTOM_NORTH_EAST_CORNER_SHAPE Lcsp;
+	FIELD w BOTTOM_SOUTH_EAST_CORNER_SHAPE Lcsp;
+	FIELD x TOP_NORTH_EAST_CORNER_SHAPE Lcsp;
+	FIELD y TOP_SOUTH_EAST_CORNER_SHAPE Lcsp;
+	FIELD z TOP_SHAPES [Lcsp;
+	METHOD <init> (Lbvj;Lbml$c;)V
+		ARG 1 baseBlockState
+		ARG 2 settings
+	METHOD a composeShape (ILcsp;Lcsp;Lcsp;Lcsp;Lcsp;)Lcsp;
+		ARG 0 i
+		ARG 1 base
+		ARG 2 northWest
+		ARG 3 northEast
+		ARG 4 southWest
+		ARG 5 southEast
+	METHOD a composeShapes (Lcsp;Lcsp;Lcsp;Lcsp;Lcsp;)[Lcsp;
+		ARG 0 base
+		ARG 1 northWest
+		ARG 2 northEast
+		ARG 3 southWest
+		ARG 4 southEast
+	METHOD a (Lcsp;Lcsp;Lcsp;Lcsp;Lcsp;I)Lcsp;
+		ARG 5 i
+	METHOD e (Lbvj;Lbgr;Lev;Lfa;)Z
+		ARG 0 state
+		ARG 1 view
+		ARG 2 pos
+		ARG 3 dir
 	METHOD j isStairs (Lbvj;)Z
 		ARG 0 state
+	METHOD m (Lbvj;Lbgr;Lev;)Lbwq;
+		ARG 0 state
+		ARG 1 view
+		ARG 2 pos
 	METHOD n (Lbvj;)Z
 		ARG 1 state
+	METHOD q getShapeIndexIndex (Lbvj;)I
```

Comment:
This is probably left over from before the flattening and called `getMetadata`. `SHAPE_INDICES` would probably be `METADATA_TO_SHAPE_INDEX`,